lib/full_dqt/old_record_mapper.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module FullDqt
+  # Given a response from new full DQP api
+  # map to old format for backwards compatability
+  class OldRecordMapper
```

Comment:
What's the plan with that? Eventually we remove the old dqt api, and we make everything work with the new format (and remove this class)?

Reply:
yea something like that. this way i don't have to change any code and the feature can be on or off. once we just have just the one there won't be a need to maintain compatibility and we can just change to use the newer structure